### PR TITLE
Add Orange County Transit Authority (OCTA) in southern California

### PR DESCRIPTION
**Pull request by n76**
Not sure if I've got this correct at all. Should be reviewed carefully before merge.

Several weeks ago I submitted the OCTA data for inclusion in Navitia and received an email showing it was accepted. While I see the transit stops on their website, I don't see the OCTA as one of the data sets when I search, so I may have the dataset ID stuff wrong in this pull request.

Signed-off-by: Tod Fitch <Tod@FitchDesign.com>

### CODE REVIEW

**Comment by n76** [comment]
Looks like many [California transit agencies included in one file](https://navitia.opendatasoft.com/explore/dataset/us-ca/information/?sort=type_file), changing pull to be for file for whole state.

File: enabler/test/de/schildbach/pte/live/CaliforniaProviderLiveTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 Tod Fitch.
```
Comment (patkan) [inline enabler/test/de/schildbach/pte/live/CaliforniaProviderLiveTest.java]: You could update the date to 2017

File: enabler/test/de/schildbach/pte/live/CaliforniaProviderLiveTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import de.schildbach.pte.CaliforniaProvider;
+import de.schildbach.pte.dto.Location;
+import de.schildbach.pte.dto.LocationType;
+import de.schildbach.pte.dto.NearbyLocationsResult;
+import de.schildbach.pte.dto.QueryDeparturesResult;
+import de.schildbach.pte.dto.SuggestLocationsResult;
+
+/**
+ * @author Stephane Berube
```
Comment (patkan) [inline enabler/test/de/schildbach/pte/live/CaliforniaProviderLiveTest.java]: You should put your name here

**Comment by n76** [comment]
Revised pull request for California added, so dropping this one.

**Pull request closed without merging**